code/src/App.js

```diff
@@ -1,12 +1,18 @@
-import React from 'react'
-import data from './data.json'
-
-console.log(data)
+import React from "react";
+import data from "./data.json";
+import { Album } from "./components/Album";
+import { Header } from "./components/Header";
 
+console.log(data);
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+      <Header />
+      <div className="album-wrapper">
+        {data.albums.items.map((item) => (
+          <Album key={item.id} item={item} artist={item.artists} />
```

Comment:
How about renaming 'item' to 'album' or 'release' or something a little more semantic?